src/replication/replication_schema_extract.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2008 Search Solution Corporation. All rights reserved by Search Solution.
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 2 of the License, or
+ *   (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ */
+
+/*
+ * replication_schema_extract.cpp
+ */
+
+#include "replication_schema_extract.hpp"
+#include "extract_schema.hpp"
+#include "network.h"
+#include "network_interface_cl.h"
+#include "unloaddb.h" // for extract_schema, extract_triggers, emit_indexes
+
+namespace cubreplication
+{
+
+  net_print_output::net_print_output (const int buffer_type, const size_t flush_size)
+    : m_buffer_type (buffer_type),
```

Comment:
It is easier to add, remove or use macros if comma is used on next line:
```suggestion
    : m_buffer_type (buffer_type)
    , m_flush_size (flush_size)
```